cli/bin.js

```diff
@@ -90,6 +90,18 @@ yarg
         type: 'boolean',
         default: false,
       });
+      yargs.option('reuse-existing-versions', {
```

Comment:
I guess this is what makes it show up in `--help`? If so disregard question #2 from earlier....

Reply:
Yeah, haven't tested tbh, but I think the `yargs` package should do this ootb

Reply:
it does